transformer_engine/pytorch/attention.py

```diff
@@ -2090,7 +2257,14 @@ def backward(ctx, dout):
                             dout_ = dout.view(-1, *dout.shape[-3:])
                         elif ctx.qkv_format == "thd":
                             q_, kv_, out_, dout_ = q, kv, out, dout
-                        aux_ctx_tensors = [softmax_lse, rng_states[cp_size - i - 1]]
+                        if ctx.fp8:
+                            aux_ctx_tensors = [
+                                softmax_lse,
+                                softmax_lse,
```

Comment:
Why do we need two `softmax_lse` here?

Reply:
FP8 kernel has two aux tensors, M and ZInV. Your code repurposed M for softmax_lse and ignored ZInv, so I just send softmax_lse to ZInv as a simple placeholder.

Reply:
Let's check if `None` works for ZInv.

Reply:
None cannot work because `Aux_CTX_Tensors` needs to be [a list of tensors](https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/pytorch/csrc/extensions/attention.cu#L924).